backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminDatasetResource.java

```diff
@@ -122,4 +152,17 @@ public void removeTable(@PathParam(TABLE_NAME) TableId tableParam) throws IOExce
 			w.send(new UpdateDataset(namespace.getDataset()));
 		}
 	}
+
+	@GET
+	@Path("tables")
+	public List<TableId> listTables(){
+		return new ArrayList<>(namespace.getDataset().getTables().keySet());
+	}
+
+	@GET
+	@Path("concepts")
+	public List<ConceptId> listConcepts(){
+		return namespace.getStorage().getAllConcepts().stream().map(Concept::getId).collect(Collectors.toList());
```

Comment:
die Concepts lassen sich nicht so wie Tables über das Dataset abrufen? Nur über den Storage?

Reply:
hab mal eine Todo in Dataset gemacht, dass wir das dann mal wenn wir Zeit haben hinzufügen